docs/integrations/public-ip.md

```diff
@@ -0,0 +1,25 @@
+# Pipes' Public IP Addresses
+
+## **Overview**
+This document provides a list of public IPs used by Pipes for outbound requests. These IPs are intended for end-users to configure whitelisting, firewall rules, and integrations with external services.
+
+## **Public IPs**
+#### **1. Load Balancer IPs**
+The following IPs are used by Pipes' load balancers. Please ensure these are whitelisted for access to our services:
+
+| **Load Balancer Name**       | **Static IP Address**     | **Ports**     |
+|------------------------------|---------------------------|---------------|
+| Main Service           | 34.86.218.179            | 80, 443       |
```

Comment:
What is `Main Service`? Is this API, or something else?

Reply:
Yeah! This is the main load balancer that redirects traffic from our application url to our services.

Reply:
Is there a better name than `Main Service`?